project

```diff
@@ -0,0 +1 @@
+../project_cbigr
```

Comment:
@maltheism what is this file ?

Reply:
@ridz1208 oh that's my project directory and for some reason it's there when it shouldn't be. I have a few project directories that I do `ln -s ../project_cbigr project` and so I can have multiple config.xml